### PR TITLE
give read more button for the specific book page

### PR DESCRIPTION
**Pull request by Saikiruthiga**
This PR includes
Change some text in fallback book image
add read more button in the specif book page for detailed description

### CODE REVIEW

File: app/app/signup/page.jsx [hunk 1]
```diff
@@ -391,6 +391,242 @@ const SignUp = () => {
...
+                },
+                "& .MuiInputLabel-root": {
+                  color: theme === "dark" ? "#bbbbbb" : "#5A4A42",
+                  fontFamily: "serif",
+                },
+                "& .MuiOutlinedInput-notchedOutline": {
+                  borderColor: theme === "dark" ? "#444444" : "#5A4A42",
+                },
+              }}
+            />{" "}
```
Comment (magdazelena) [inline app/app/signup/page.jsx:515]: are these supposed to be here? :)

File: app/app/signup/page.jsx [hunk 2]
```diff
@@ -391,6 +391,242 @@ const SignUp = () => {
...
+                mt: 8, // Add spacing between the buttons
+                color: theme === "dark" ? "#bbbbbb" : "#5A4A42",
+                fontFamily: "jacques Francois",
+                fontSize: "16px",
+              }}
+              onClick={() => router.push("/")}
+            >
+              Back to Home
+            </Button>
+          </Container>{" "}
```
Comment (magdazelena) [inline app/app/signup/page.jsx:617]: in many places

File: app/app/signup/page.jsx [hunk 3]
```diff
@@ -400,6 +636,7 @@ const SignUp = () => {
             />
         </AppLayoutContainer>
     );
+>>>>>>> origin/main
```
Comment (magdazelena) [inline app/app/signup/page.jsx:639]: oops some leftovers after rebase!

File: app/app/signup/page.jsx [hunk 4]
```diff
@@ -391,6 +391,242 @@ const SignUp = () => {
...
+      <ErrorModal
+        isOpen={error.isOpen}
+        onClose={hideError}
+        message={error.message}
+        title={error.title}
+        severity={error.severity}
+      />
+    </AppLayoutContainer>
+  );
+=======
```
Comment (magdazelena) [inline app/app/signup/page.jsx:629]: and here

**Review by magdazelena — COMMENTED** [review]
some details!

File: app/app/signup/page.jsx (hunk 4 above)
Comment (Saikiruthiga) [reply app/app/signup/page.jsx:629]: Yes, I messed with merge conflicts. I realized and fixed everything. Thank you for your time

**Pull request merged**